src/goto-instrument/loop_utils.cpp

```diff
@@ -107,3 +108,41 @@ void get_modifies(
     }
   }
 }
+
+// TODO handle aliasing at all
+void get_uses(
```

Comment:
I would rename this to something more verbose, say "get_used_symbols", and also comment that you do include symbols whose address is taken.